client/my-sites/plugins/main.jsx

```diff
@@ -61,7 +61,7 @@ export default React.createClass( {
 	mixins: [ URLSearch, PluginNotices ],
 
 	getInitialState() {
-		return this.getPluginsState( this.props );
+		return Object.assign( {}, this.getPluginsState( this.props ), { bulkManagement: [] } );
```

Comment:
since `this.getPluginsState` is creating a new object in each return, the first param (`{}`) is not required